gnmi_cli_py/py_gnmicli.py

```diff
@@ -403,19 +405,23 @@ def gen_request(paths, opt, prefix):
     yield mysubreq
 
 
-def subscribe_start(stub, options, req_iterator):
+def subscribe_start(stub, options, req_iterator, exit_early):
   """ RPC Start for Subscribe reqeust
   Args:
       stub: (class) gNMI Stub used to build the secure channel.
       options: (dict) Command line argument passed for subscribe reqeust.
       req_iterator: gNMI Subscribe Request from gen_request.
+      exit_early: create TCP connections with gNMI server without explicitly 
+                  closeing it to simulate memory spike on server side.
   Returns:
       Start Subscribe and printing response of gNMI Subscribe Response.
   """
   metadata = [('username', options['username']), ('password', options['password'])]
   max_update_count = options["update_count"]
   try:
       responses = stub.Subscribe(req_iterator, options['timeout'], metadata=metadata)
+      if exit_early:
+         return 
```

Comment:
>return [](http://example.com/codeflow?start=9&length=6)

weird indentation

Reply:
Initially I followed the indentation style in this file which used mixed 2 white spaces and 4 white spaced as an indentation. I will update all indentation in this file.

Reply:
Sorry for misleading. I am not asking regarding original code, and only ask you focus on your own code, which indent by 3 blanks.

Since this is a critical feature PR, you'd better keep it small. Could you revert back all the non-critical format change?

Reply:
Updated.